R/parse-body.R

```diff
@@ -254,45 +333,44 @@ parser_multi <- function() {
           x$content_type <- getContentType(tools::file_ext(x$filename))
         }
       }
+      x$parsers <- parsers
       parse_raw(x)
     })
   }
+  return(make_parsers(f, fixed = "multipart/form-data"))
 }
 
+#' @describeIn parsers All parsers
+#' @export
+parser_all <- function() {
+  parsers <- .globals$parsers
+  parsers$all <- NULL
```

Comment:
Is this line needed? 

```suggestion
  parsers$all <- NULL
```

Reply:
Yes, otherwise it does a recursive call to the `all` parsers and you end up with a C error stack.